server/src/schema/migrations/1746636476623-DropExtraIndexes.ts

```diff
@@ -0,0 +1,17 @@
+import { Kysely, sql } from 'kysely';
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await sql`ALTER DATABASE "immich" SET search_path TO "$user", public, vectors`.execute(db);
+  await sql`ALTER TABLE "naturalearth_countries" DROP CONSTRAINT IF EXISTS "PK_21a6d86d1ab5d841648212e5353";`.execute(db);
+  await sql`ALTER TABLE "naturalearth_countries" DROP CONSTRAINT IF EXISTS "naturalearth_countries_pkey";`.execute(db);
+  await sql`ALTER TABLE "naturalearth_countries" ADD CONSTRAINT "naturalearth_countries_pkey" PRIMARY KEY ("id") WITH (FILLFACTOR = 100);`.execute(db);
```

Comment:
The key is called `PK_21a6d86d1ab5d841648212e5353` in the migration, but `naturalearth_countries_pkey` after the geodata import has completed. Renaming it in the migration will reconciles this discrepancy.